index.html

```diff
@@ -47,31 +50,31 @@ <h3 class="my-info-skill">I’am a Software Developer</h3>
       
         <ul class="social-media-links">
         <li class="link-items">
```

Comment:
- [ **OPTIONAL** ] **IMPORTANT:** Kindly make sure you use the correct `tag` for accessibility as it gives this error: **Heading elements are not in a sequentially-descending order** on the text `I'm a Software Developer`(line 47). I think you can use `h2` instead of `h3` and the problem will be solved.